packages/docs/docs/agent/index.md

```diff
@@ -173,6 +173,12 @@ npm run agent <base_url> <client_id> <client_secret> <agent_id>
 
 Keep the terminal open so that you can see the logs. At this point you can start sending messages to the agent.
 
+## Supported Protocols
+
+- [HL7 v2](https://www.hl7.org): A low-level transfer protocol for HL7 content.
+- [DICOM C-Store](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html): A protocol used for storing metadata about medical images.
+- [ASTM](https://www.astm.org/): Used to transfer data between clinical instruments and computer systems.
```

Comment:
We may want to flag this as "experimental" or "alpha"

Reply:
Do we want to wait until we add it back to the agent before we list it as supported? Potential documentation drift

Reply:
@ThatOneBro - Is this dependent on the serialport change we made? I thought the protocol and the connection was independent. 

@ksmith94 - I support flagging as "alpha"

Reply:
This has been flagged as "alpha". Let me know if you want to remove it or make any other changes.